client/Dockerfile-test

```diff
@@ -4,6 +4,7 @@ FROM simpletaskmanager/stm-client-base:1.1.0-dev as builder
 COPY ./ /stm-client/
 WORKDIR /stm-client/
 
+RUN NODE_OPTIONS="--max_old_space_size=4096" npm install @angular/localize
```

Comment:
Why is this necessary? The @angular/localize package would be installed via the package.json, wouln't it?